ztools/xssh/common.go

```diff
@@ -25,6 +25,8 @@ const (
 )
 
 // supportedCiphers specifies the supported ciphers in preference order.
+// These are *not* the supported ciphers, they are the default ciphers.
+// The all supported ciphers are listed in the ciphers.go -> cipherModes map
 var supportedCiphers = []string{
```

Comment:
It seems easier for me to just do it and show you, than keep attempting to explain this over Github review.